quantuminspire/util/configuration.py

```diff
@@ -171,6 +171,8 @@ def write_settings_to_file(self) -> None:
     @staticmethod
     async def _fetch_team_member_id(host: str, access_token: str) -> int:
         config = Configuration(host=host, access_token=access_token)
+        # Wait for sometime as we might get token not yet valid error due to clock sync problem between servers
+        await asyncio.sleep(3)
```

Comment:
can't we decode the token, check from which timestamp the token is valid and wait until that time?

Reply:
indeed that is a better idea. Implemented that